app/controllers/atmosphere/api/v1/clew_controller.rb

```diff
@@ -22,8 +22,10 @@ def appliance_instances
         end
 
         def appliance_types
-          appliance_types = @appliance_types.active.includes(:tenants).references(:tenants).
-              includes(:appliance_configuration_templates).references(:appliance_configuration_templates).order(:id)
+          appliance_types = @appliance_types.active.
+              includes(:tenants, :appliance_configuration_templates).
+              references(:tenants, :appliance_configuration_templates).
+              where(atmosphere_tenants: { id: current_user.tenants.active }).order(:id)
```

Comment:
Line is too long. [87/80]<br>Align the operands of an expression in an assignment spanning multiple lines.